docs/upgrade.html

```diff
@@ -73,7 +73,13 @@ <h5><a id="upgrade_311_notable" href="#upgrade_311_notable">Notable changes in 3
         via Connect worker and/or connector configuration. Connect may enable idempotent producers
         by default in a future major release.</li>
     <li>Kafka has replaced log4j and slf4j-log4j12 with reload4j and slf4j-reload4j due to security concerns.
-        More information can be found at <a href"https://reload4j.qos.ch">reload4j</a>.</li>
+        This only affects modules that specify a logging backend (<code>connect-runtime</code> and <code>kafka-tools</code> are two such examples).
```

Comment:
Good point! Since this is actually a cherry-pick, I would propose to get this merged to be consistent with the source of this cherry-pick and then we can look into enumerating the modules. WDYT?

Reply:
SGTM.